src/main/java/ladder/domain/Name.java

```diff
@@ -1,24 +1,52 @@
 package ladder.domain;
 
+import java.util.Objects;
 import java.util.regex.Pattern;
 
 public class Name {
   private final String name;
+  private final Pattern pattern = Pattern.compile(INVALID_PATTERN_REGEX);
   private final static String INVALID_PATTERN_REGEX = "^[가-힣a-zA-Z]*$";
```

Comment:
```suggestion
  private final static String INVALID_PATTERN_REGEX = "^[가-힣a-zA-Z]*$";
  private final static Pattern PATTERN = Pattern.compile(INVALID_PATTERN_REGEX);
```
클래스 상수로 선언해도 되겠어요. 🙂